app/policies/roles/site/institution.json.erb

```diff
@@ -0,0 +1,66 @@
+[
+  {
+    "name": "Site <%=  %> Admin",
+    "policy": {
+      "statement": [
+        {
+          "resource": "site/<%= site_id %>",
+          "action": [
+            "site:read",
+            "site:update",
+            "site:delete"
+          ],
```

Comment:
I didn't include the final correct JSON files because this were "to be defined". This PR is just for the funcionality, a separate commit could provide the correct JSON files.